README.md

```diff
@@ -358,6 +358,7 @@ irb(main):002> a.first. # Completes Integer methods
 - `NO_COLOR`: Assigning a value to it disables IRB's colorization.
 - `IRB_USE_AUTOCOMPLETE`: Setting it to `false` disables IRB's autocompletion.
 - `IRB_COMPLETOR`: Configures IRB's auto-completion behavior, allowing settings for either `regexp` or `type`.
+- `IRB_COPY_COMMAND`: Overrides the default program used to interface with the system clipboard.
```

Comment:
Does the documentation need to be updated anywhere other than here and the man page?